### ISSUE TITLE
Additional synonyms to expand support for Ensembl genes cross-references

### ISSUE THREAD

**Issue by dhimmel** [biopragmatics/bioregistry#294]
In https://github.com/related-sciences/ensembl-genes/pull/15, we use `bioregistry.normalize_curie` on the Ensembl provided cross-references for genes. Several of the prefixes used by Ensembl aren't supported by Bioregistry. See below the rows where `bioregistry_prefix` equals `None`.

xref_source | database | count | bioregistry_prefix
-- | -- | -- | --
ArrayExpress | homo_sapiens_core_105_38 | 68005 | arrayexpress
DBASS3 | homo_sapiens_core_105_39 | 19 | None
DBASS5 | homo_sapiens_core_105_40 | 13 | None
ENS_LRG_gene | homo_sapiens_core_105_41 | 1324 | None
EntrezGene | homo_sapiens_core_105_42 | 29122 | ncbigene
HGNC | homo_sapiens_core_105_43 | 44070 | hgnc
MIM_GENE | homo_sapiens_core_105_44 | 18283 | None
MIM_MORBID | homo_sapiens_core_105_45 | 7391 | None
RFAM | homo_sapiens_core_105_46 | 5080 | rfam
Reactome_gene | homo_sapiens_core_105_47 | 136408 | None
Uniprot_gn | homo_sapiens_core_105_48 | 86144 | None
WikiGene | homo_sapiens_core_105_49 | 29122 | None
miRBase | homo_sapiens_core_105_50 | 2113 | mirbase
ArrayExpress | rattus_norvegicus_core_105_72 | 30559 | arrayexpress
EntrezGene | rattus_norvegicus_core_105_73 | 23564 | ncbigene
MGI | rattus_norvegicus_core_105_74 | 672 | mgi
RFAM | rattus_norvegicus_core_105_75 | 2029 | rfam
RGD | rattus_norvegicus_core_105_76 | 27631 | rgd
Reactome_gene | rattus_norvegicus_core_105_77 | 76351 | None
Uniprot_gn | rattus_norvegicus_core_105_78 | 15124 | None
WikiGene | rattus_norvegicus_core_105_79 | 23564 | None
miRBase | rattus_norvegicus_core_105_80 | 428 | mirbase

Source notebooks for [human](https://github.com/related-sciences/ensembl-genes/blob/13d405e5026d9109b8ada7328d59b4c4d7bd7c3c/ensembl_genes_eda.ipynb) and [rat](https://github.com/related-sciences/ensembl-genes/blob/8f134725bb1a8989b676a06c38928d9af2b3f71c/ensembl_genes_eda.ipynb).

Going through the unmapped prefixes, we could add the following synonyms:

- `WikiGene` --> `wikigenes`
- possibly `Reactome_gene` --> `reactome`. Example ID is `R-HSA-9664417`, which looks like a pathway not a gene, so not sure why ensembl puts a `_gene` suffix.
- `Uniprot_gn` --> `uniprot`, example id `Q5TFM2`
- MIM_GENE --> mim example 134370, https://omim.org/entry/134370
- MIM_MORBID --> mim, example 126700, resolve to https://omim.org/entry/126700

Not sure whether these synonyms should be added to Bioregistry, or we should map them locally if they are too dataset specific to be considered general synonyms.

**Comment by cthoyt** [comment]
I'm happy to entertain adding these synonyms. The reactome one is quite strange, I'm not sure what ensembl is doing. I'd rather this gets an upstream fix unless it's impossible to petition them to update. I'm curious if there's some insight on what `_gn` means in the uniprot reference too.

Are you familiar with DBASS3, DBASS5, or ENS_LRG_gene? I'd also be motivated to get new prefix requests for those

**Comment by dhimmel** [comment]
> I'd rather this gets an upstream fix

Agreed, although it might be difficult. Tagging @michalszpak from Ensembl who's helped us with other questions: Is there interest in changing the values in `external_db.db_name` to more standard resource identifiers? Or perhaps adding a new column for the standard database prefix?

Also what does `_gn` in `Uniprot_gn` stand for? I'm guessing it stands for _gene_, but that doesn't make a ton of sense since the cross-reference is actually to a uniprot protein. Similar with `Reactome_gene`, where the cross-references are to pathways and not genes.

> Are you familiar with DBASS3, DBASS5, or ENS_LRG_gene?

- ENS_LRG_gene --> lrg, http://bioregistry.io/registry/lrg
- DBASS3 adn DBASS5 refer to http://www.dbass.org.uk/, but don't yet understand the IDs and how they'd expand to URLs

**Comment by cthoyt** [comment]
@dhimmel do you have a follow-up for this?
